ccf_transaction_processor/scripts/start_cchost.sh

```diff
@@ -23,5 +23,5 @@ EFILE="${F_SERVICEHOME}/logs/error.log"
 OFILE="${F_SERVICEHOME}/logs/output.log"
 
 cd ${F_SERVICEHOME}/run
-${CCHOST} --config ${F_SERVICEHOME}/etc/cchost.toml --node-pid-file ${F_SERVICEHOME}/run/cchost.pid > $OFILE 2> $EFILE &
+${CCHOST} --san dNSName:pdo-tp-ccf --config ${F_SERVICEHOME}/etc/cchost.toml --node-pid-file ${F_SERVICEHOME}/run/cchost.pid > $OFILE 2> $EFILE &
```

Comment:
this has to be changed to either PDO_HOSTNAME or the host name from the PDO_LEDGER_URL. As it stands, this makes this script work PRECISELY in the CI and nowhere else.